src/Microsoft.Restier.Publishers.OData/Routing/HttpConfigurationExtensions.cs

```diff
@@ -129,5 +128,29 @@ public static class HttpConfigurationExtensions
             conventions.Insert(index + 1, new RestierRoutingConvention(apiFactory));
             return conventions;
         }
+
+        private static IEdmModel GetModel(ApiBase api)
+        {
+            // Here await is not used because if method MapRestierRoute is mapped async,
+            // Then during application starts, the http service initialization may complete first before this method call is complete.
+            // Then all request will fail, and this happen for some test cases before when get model takes long time.
+            IEdmModel model;
+            try
+            {
+                model = api.GetModelAsync().Result;
+            }
+            catch (Exception e)
```

Comment:
better use AggreateException and no need to check if it contains an InnerException (because it always has)